crates/nu-command/src/system/run_external.rs

```diff
@@ -80,23 +80,27 @@ impl Command for External {
 
         let paths = nu_engine::env::path_str(engine_state, stack, call.head)?;
 
-        // On Windows, the user could have run the cmd.exe built-in "assoc" command
-        // Example: "assoc .nu=nuscript" and then run the cmd.exe built-in "ftype" command
-        // Example: "ftype nuscript=C:\path\to\nu.exe '%1' %*" and then added the nushell
-        // script extension ".NU" to the PATHEXT environment variable. In this case, we use
-        // the which command, which will find the executable with or without the extension.
-        // If it "which" returns true, that means that we've found the nushell script and we
-        // believe the user wants to use the windows association to run the script. The only
-        // easy way to do this is to run cmd.exe with the script as an argument.
-        let potential_nuscript_in_windows = if cfg!(windows) {
-            // let's make sure it's a .nu script
+        // On Windows, ther user could have run the cmd.exe built-in commands "assoc"
+        // and "ftype" to create a file association for an arbitrary file extension.
+        // They then could have added that extenstion to the PATHEXT environment variable.
+        // In this case we use the which command, which will find the executable with
+        // or without the extension. If "which" returns true, that means that we've
+        // found the script and we believe the user wants to use the windows association
+        // to run the script. The only easy way to do this is to run cmd.exe with the
+        // script as an argument. File extensions of .COM, .EXE, .BAT, and .CMD
+        // are ignored because Windows can run those files directly. PS1
```

Comment:
You get my drift here. I just wanted to tweak the wording.
```suggestion
        // are ignored because Windows can run those files directly. PS1 files are also ignored and that extension is handled in a separate block below.
```

Reply:
I included this change with the other comment change in commit `19e2751`